README.md

```diff
@@ -73,6 +91,10 @@ To do that, put files into the correct directories according to this table.
 
 Doing this will make niri appear in GDM and, presumably, other display managers.
 
+### NixOS
+
+For NixOS users, check out https://github.com/sodiboo/niri-flake
```

Comment:
Could you keep this inside the tip above for now? I think it makes more sense, because people are likely first interested in "normal" installation, which is why I put that closer to the top.